cosmwasm/packages/wormhole-bindings/src/query.rs

```diff
@@ -0,0 +1,42 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Binary, CustomQuery, Empty};
+
+#[cw_serde]
+pub struct Signature {
+    // The index of the observer in the observer set.
+    pub index: u16,
+
+    // The signature.
+    pub signature: Binary,
```

Comment:
Given that this is expected to have a specific structure, might be worth making the type more precise. Unfortunately I've seen different libraries/ecosystems use different permutations of the 65 byte (v,r,s) triple, so the signature bytes can't be treated abstract in a portable way.

Reply:
The 65 byte (v,r,s) serialization is the most common I've seen.  Unless we're commonly parsing out v,r,s instead of passing it transparently to a library, I think keeping the blob would be the most simple.

Reply:
Done.